packages/flutter_tts/example/tizen/tizen-manifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest package="org.tizen.flutter_tts_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
-    <profile name="common"/>
+    <profile name="wearable"/>
```

Comment:
I think this change was resulted by https://github.com/flutter-tizen/flutter-tizen/pull/164. You should manually reset or exclude from staged changes since this plugin supports both wearable and TV profiles.